lib/Http/RequestHelper.php

```diff
@@ -187,4 +195,22 @@ public function anonymizeIp(string $ip)
 
         return $aip;
     }
+
+    /**
+     * @param string $uri
+     * @return Request
+     */
+    public function createRequestWithContext($uri = '/') {
+        $request =  Request::create(
+            $this->requestContext->getScheme().'://'.$this->requestContext->getHost().$this->requestContext->getBaseUrl().$uri,
```

Comment:
We should consider non-standard ports here: 

```php
if ('http' === $scheme && 80 !== $this->requestContext->getHttpPort()) {
    $port = ':'.$this->requestContext->getHttpPort();
} elseif ('https' === $scheme && 443 !== $this->requestContext->getHttpsPort()) {
    $port = ':'.$this->requestContext->getHttpsPort();
}
```

Reply:
done!